src/testdrive/src/action/sql.rs

```diff
@@ -139,16 +139,24 @@ impl Action for SqlAction {
                     if retry_state.i != 0 {
                         println!();
                     }
-                    println!("rows match; continuing");
+                    println!(
+                        "rows match; continuing at ts {}",
+                        SystemTime::now()
+                            .duration_since(SystemTime::UNIX_EPOCH)
```

Comment:
This uses the non-monotonic clock to obtain unix timestamps  . If the unwrap fires in practice I will have to replace it with a monotonic clock.